rfcs/0002-branch-generation.md

```diff
@@ -0,0 +1,76 @@
+# Branch Generation
+
+The goal of this RFC is to define how Solana generates branches.
```

Comment:
Solana doesn't generate branches. And if "branches" are different than "forks", you'll want to explain that here.